jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/ConfigHelper.java

```diff
@@ -211,6 +212,7 @@ private static void verifyImageNames(List<ImageConfiguration> ret) {
             BuildConfiguration buildConfiguration = image.getBuildConfiguration();
             if (buildConfiguration != null &&  buildConfiguration.isDockerFileMode()) {
                 log.info("Using Dockerfile: %s", buildConfiguration.getDockerFile().getAbsolutePath());
+                log.info("Using Docker Context Directory: %s", buildConfiguration.getAbsoluteContextDirPath(jKubeConfiguration.getSourceDirectory(), jKubeConfiguration.getBasedir().getAbsolutePath()));
```

Comment:
Aren't these directories in JavaProject?

I remember in the early days these fields were scattered all around and we managed to reduce them to just the JKubeConfiguration and the JavaProject instances.
What's the issue in this case?

Reply:
If the homonymous fields have the same purpose, one set should be deprecated. If not, the undocumented, should be. This way we would avoid situations like the current were there's confusion about what should be used

Reply:
I can see `jKubeConfiguration.getBaseDir()` returns JavaProject's  base directory. But I'm not able to find any `sourceDirectory` equivalent in JavaProject

Reply:
```suggestion
                log.info("Using Docker context directory: %s", buildConfiguration.getAbsoluteContextDirPath(jKubeConfiguration.getSourceDirectory(), jKubeConfiguration.getBasedir().getAbsolutePath()));
```

Reply:
> I can see jKubeConfiguration.getBaseDir() returns JavaProject's base directory. But I'm not able to find any sourceDirectory equivalent in JavaProject

OK, so this one is not duplicate, it just retrieves javaProject.baseDirectory

However, `outputDirectory` is still defined in both places. We'll need to investigate and deprecate, rename or proper document them.